### PR TITLE
Bring back dev branch, re-enable storage emulator tests

### PR DESCRIPTION
**Pull request by adamchester**
cc @robdmoore 

P.s. I see I have editor tab/space issues here, I'll normalise them separately in `dev` branch.

### CODE REVIEW

**Comment by adamchester** [comment]
OK this seems to be ready for merge into `dev`. We might have to live with some instability in the CI, as I noticed the first build [failed with a strange error related to local storage emulator](https://ci.appveyor.com/project/serilog/serilog-sinks-azuretablestorage/build/25).

**Comment by robdmoore** [comment]
This looks good. Will the emulator tests run on app veyor or is that still a todo?

**Comment by adamchester** [comment]
@robdmoore they work OOTB on the vs2015 image, surprisingly :+1:

File: src/Serilog.Sinks.AzureTableStorage/project.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+﻿{
+    "version": "2.0.1-*",
+    "description": "Serilog event sink that writes to Azure Table Storage over HTTP.",
+    
+    "authors": [ "Robert Moore" ],
+    "packOptions": {
+        "tags": [ "serilog", "logging", "azure" ],
+        "projectUrl": "http://serilog.net",
+        "licenseUrl": "http://www.apache.org/licenses/LICENSE-2.0",
+        "iconUrl": "http://serilog.net/images/serilog-configuration-nuget.png"
```
Comment (nblumhardt) [inline src/Serilog.Sinks.AzureTableStorage/project.json]: I just noticed on NuGet this is wrong; should be `serilog-sinks-nuget.png`.

File: src/Serilog.Sinks.AzureTableStorage/project.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+﻿{
+    "version": "2.0.1-*",
+    "description": "Serilog event sink that writes to Azure Table Storage over HTTP.",
+    
+    "authors": [ "Robert Moore" ],
+    "packOptions": {
+        "tags": [ "serilog", "logging", "azure" ],
+        "projectUrl": "http://serilog.net",
```
Comment (nblumhardt) [inline src/Serilog.Sinks.AzureTableStorage/project.json]: https://github.com/serilog/serilog-sinks-azuretablestorage ?

**Comment by adamchester** [comment]
thanks @nblumhardt - they should be fixed now

**Comment by adamchester** [comment]
Merge me please :smiley_cat:

**Pull request merged**